core/domain/state_domain.py

```diff
@@ -281,8 +281,10 @@ def __init__(
             interaction_id: str. The interaction id.
             answer_is_exclusive: bool. True if is the only correct answer;
                 False if is one of possible answer.
-            correct_answer: str. The correct answer; this answer enables the
-                learner to progress to the next card.
+            correct_answer: list(set) or str. The type of correct_answer is
+                determined by the value of BaseInteraction.answer_type.
+                The correct answer;
```

Comment:
I don't understand this sentence. I think you may want to reword as:

correct_answer: *. The correct answer; this answer enables the learner to progress to the next card. (The type of the correct_answer is determined by ...)

Reply:
done